docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -191,7 +191,120 @@ adding a Channel Binder implementation
 (e.g. `spring-cloud-starter-stream-rabbit` for RabbitMQ or
 `spring-cloud-starter-stream-kafka` for Kafka). This will
 automatically turn your app into a producer of messages with payload
-type `Spans`. 
+type `Spans`.
+
+== Customizations
+
+Thanks to the `SpanInjector` and `SpanExtractor` you can customize the way spans
+are created and propagated.
+
+Essentially with Spring Cloud Span there are two ways span tracing data can be passed to
+ a process.
+
+ * via Spring Integration
+ * via HTTP
+
+=== Spring Integration
+
+For Spring Integration these are the beans responsible for creation of a Span from a `Message`
```

Comment:
before sections like this. overview in a sentence or two how propagation works. Forgive the rough sentences.

ex. For HTTP, propagation is via http headers. For example, span ids are extracted from Zipkin-compatible (B3) headers, to start or join an existing trace. Trace information is injected into any outbound requests so the next hop can extract them.